### PR TITLE
wordy: align with specification per #530

### LINKED ISSUE #530: wordy: update to most recent canonical specification
Recently a `.meta/tests.toml` file was introduced to each exercise on this track.
The purpose being to show how an exercise aligns (or does not align)  to the exercise's `canonical-data.json` (in the [problem-specifications](https://github.com/exercism/problem-specifications/) repo), by comparing the UUID assigned to each test case.

See the [canonical-data-syncer](https://github.com/exercism/canonical-data-syncer) for how to check the exercises for this.

This has shown that the wordy exercise has missing or out of date tests.

The request for this issue is to assign the issue to yourself and then (having read our [CONTIBUTING.md](https://github.com/exercism/c/blob/master/docs/CONTRIBUTING.md)) update the exercise such that:

- Existing test names are correct
- Existing tests match the requirements of the related `canonical-data.json`
- Missing tests are added
  - If needed, the example implementation is adjusted to fulfill the added test cases
- Update the related `tests.toml` file to add the UUIDs that correspond to the updated tests

See the following past PR for an example #493

### PR DESCRIPTION
**Pull request by wolf99**
Fixes #530 

Interesting to see the x-macros in the `example.c`. I think it's the only "real-world"  use of them that I have seen, that does not involve enumerations, for a very long time!

### CODE REVIEW

**Comment by ryanplusplus** [comment]
> Interesting to see the x-macros in the `example.c`. I think it's the only "real-world" use of them that I have seen, that does not involve enumerations, for a very long time!

I use x-macros heavily in in embedded development at my day job for far more than enumerations. I wish I could share.

File: exercises/wordy/test/test_wordy.c [hunk 1]
```diff
@@ -180,26 +191,87 @@ static void test_answer_non_math_question(void)
...
+static void test_reject_two_numbers_in_a_row(void)
+{
+   TEST_IGNORE();
+   const char *question = "What is 1 plus plus 2 1?";
+   int result;
+   bool success = answer(question, &result);
+   TEST_ASSERT_FALSE(success);
+}
+
+static void test_reject_postix_notation(void)
```
Comment (ryanplusplus) [inline exercises/wordy/test/test_wordy.c:230]: This should be `postfix`, right?

Comment (ryanplusplus) [reply exercises/wordy/test/test_wordy.c:230]: Yep, I checked.

Comment (ryanplusplus) [inline exercises/wordy/test/test_wordy.c:230]:
```suggestion
static void test_reject_postfix_notation(void)
```

File: exercises/wordy/test/test_wordy.c [hunk 2]
```diff
@@ -180,26 +191,87 @@ static void test_answer_non_math_question(void)
...
+   RUN_TEST(test_multiple_multiplication);
+   RUN_TEST(test_addition_and_multiplication);
+   RUN_TEST(test_multiple_division);
+   RUN_TEST(test_unknown_operation);
+   RUN_TEST(test_non_math_question);
+   RUN_TEST(test_reject_a_problem_missing_an_operand);
+   RUN_TEST(test_reject_problem_with_no_operands_or_operators);
+   RUN_TEST(test_reject_two_operations_in_a_row);
+   RUN_TEST(test_reject_two_numbers_in_a_row);
+   RUN_TEST(test_reject_postix_notation);
```
Comment (ryanplusplus) [inline exercises/wordy/test/test_wordy.c:273]:
```suggestion
   RUN_TEST(test_reject_postfix_notation);
```

**Review by ryanplusplus — APPROVED** [review]

File: exercises/wordy/test/test_wordy.c (hunk 1 above)
Comment (wolf99) [reply exercises/wordy/test/test_wordy.c]: Well caught!

**Pull request merged**